Interview/ConductingATechnicalInterview.md

```diff
@@ -0,0 +1,24 @@
+Outline:
+
+How to Conduct a Technical Interview
+====================================
+
+### Remember what You Want to Achieve
+Best possible outcome from an interview is a new hire who makes great contributions to your organisation, your team and thrives as a member in your team. In this section I will try to say something about the mindset required to interview a candidate successfully.
+
+### Be prepared
+To not prepare for an interview is simply rude. How to best prepare does, however, depend on the kind of interview.
+
+### Make the Candidate Feel Welcome
+Discuss what we can do to make a probably nervous candidate feel welcome and respected.
+
+### Look for what the Candidate can bring
+Diverse teams are well known to be more productive but at the same time a new hire should enough about the technology we use for their intended level. How can one think about structuring an interview to learn something about what the candidate brings that we don't have already while not forgetting to ask about essential skills.
+
+###  Let the Candidate have the Floor
+- It is not your interview
+- It is not your chance to take a step forward in your career
+- Showing off your skills to a candidate is never going to earn you a pay rise or promotion. Save your boasting for your next one-to-one with your line manager.
```

Comment:
imo this sounds too aggressive, can we rephrase it? also these three items can be merged into one.

```
- be humble and let the candidate show their best rather than showing yours
```

Reply:
@ilyapuchka In this outline I am hoping to get feedback on which topics to discuss. Of course everyone will also be allowed to have an input on formulations in the final text. That said, I have noticed your point and will try to use less aggressive language in the article.

Reply:
This is just the outline, not the actual article.

Reply:
👍